src/plugins/plugin.filler.js

```diff
@@ -232,47 +233,120 @@ function isDrawable(point) {
 	return point && !point.skip;
 }
 
-function drawArea(ctx, curve0, curve1, len0, len1) {
+function fillPointsSets(ctx, curve0, curve1, len0, len1, area, pointSets) {
 	var i, cx, cy, r;
+	var fillAreaPointsSet = [];
```

Comment:
we typically put initialized variables above un-initialized variables, so I'd put the three new lines above the existing `var i, cx, cy, r;` line